core/src/test/scala/org/apache/spark/sql/delta/DeltaSuite.scala

```diff
@@ -926,7 +926,26 @@ class DeltaSuite extends QueryTest
       )
     }
   }
-
+  test("get touched files for update, delete and merge") {
+    withTempDir { dir =>
+      val directory = new File(dir, "test with space")
+      val df = Seq((1, 10), (2, 20), (3, 30), (4, 40)).toDF("key", "value")
+      val writer = df.write.format("delta").mode("append")
+      writer.save(directory.getCanonicalPath)
+      spark.sql(s"UPDATE delta.`${directory.getCanonicalPath}` SET value = value + 10")
+      spark.sql(s"DELETE FROM delta.`${directory.getCanonicalPath}` WHERE key = 4")
+      val inbound = Seq((3, 30)).toDF("key", "value").createOrReplaceTempView("inbound")
+      var mergeInto = s"MERGE INTO delta.`${directory.getCanonicalPath}` AS base "
+      val inboundDF = "USING inbound "
+      val mergeOn = "ON base.key = inbound.key "
+      val whenMatched = "WHEN MATCHED THEN UPDATE SET base.value = base.value+inbound.value"
+      spark.sql(mergeInto + inboundDF  + mergeOn  + whenMatched)
+      checkAnswer(
+        spark.read.format("delta").load(directory.getCanonicalPath),
+        Seq((1, 20), (2, 30), (3, 70)).toDF("key", "value")
+      )
+    }
+  }
```

Comment:
nit: missing an empty line between tests.

Reply:
noted and added in